File: cases/taint/ruby/map-iteration-positive/map_iteration.rb
def dfb_source # DFB-SOURCE: map-iteration-input
  "tainted"
end

def dfb_sink(value) # DFB-SINK: map-iteration-sink
end

def run
  records = {}
  records["record"] = dfb_source # DFB-WITNESS: map-iteration-store
  records.each do |key, value|
    dfb_sink(value)
  end
end

File: cases/taint/python/map-iteration-positive/map_iteration.py
def dfb_source():  # DFB-SOURCE: map-iteration-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: map-iteration-sink
    pass


def run():
    records = {}
    records["record"] = dfb_source()  # DFB-WITNESS: map-iteration-store
    for key, value in records.items():
        dfb_sink(value)
